### PR TITLE
Adding comment about default appsettings impact

### PR DESCRIPTION
**Pull request by kirankumarkolli**
adding comment about default appsettings impact

### CODE REVIEW

File: Microsoft.Azure.Cosmos/src/CosmosClient.cs [hunk 1]
```diff
@@ -104,6 +104,8 @@ static CosmosClient()
 
             // V3 always assumes assemblies exists
             // Shall revisit on feedback
+            // NOTE: Native ServiceInteropWrapper.AssembliesExist has appsettings depenency which are proofed for CTL (native dll entry) scenarios.
```
Comment (ealsur) [inline Microsoft.Azure.Cosmos/src/CosmosClient.cs]:
```suggestion
            // NOTE: Native ServiceInteropWrapper.AssembliesExist has appsettings dependency which are proofed for CTL (native dll entry) scenarios.
```

Comment (j82w) [reply Microsoft.Azure.Cosmos/src/CosmosClient.cs]: Should we add what the settings are?

File: Microsoft.Azure.Cosmos/src/CosmosClient.cs [hunk 2]
```diff
@@ -104,6 +104,8 @@ static CosmosClient()
 
             // V3 always assumes assemblies exists
             // Shall revisit on feedback
+            // NOTE: Native ServiceInteropWrapper.AssembliesExist has appsettings dependency which are proofed for CTL (native dll entry) scenarios.
```
Comment (j82w) [inline Microsoft.Azure.Cosmos/src/CosmosClient.cs]: Should we add the app setting name and values so we don't have to search through code everytime?

Comment (kirankumarkolli) [reply Microsoft.Azure.Cosmos/src/CosmosClient.cs]: Its transient and relfects ony currents state. New code can accumulate as well.

**Pull request merged**